i18n/en/docusaurus-plugin-content-docs/current.json

```diff
@@ -7,9 +7,13 @@
     "message": "What is YunoHost?",
     "description": "The label for category What is YunoHost? in sidebar admin"
   },
-  "sidebar.admin.category.Self-hosting": {
-    "message": "Self-hosting",
-    "description": "The label for category Self-hosting in sidebar admin"
+  "sidebar.admin.category.🚀 Get started": {
```

Comment:
Is that actually valid in JSON? 😲

Reply:
I guess so? I did not edit this file by myself: it has been automatically generated by the write-translations script IIRC